lib/iris/tests/unit/fileformats/netcdf/test__load_aux_factory.py

```diff
@@ -109,14 +109,46 @@ def test_formula_terms_p0_bounded(self):
                 'coordinate {!r} bounds.'.format(coord_p0.name())
             self.assertEqual(msg, str(warn[0].message))
 
+    def _check_no_delta(self):
+        # Check cube.add_aux_coord method.
+        self.assertEqual(self.cube.add_aux_coord.call_count, 0)
+        # Check cube.add_aux_factory method.
+        self.assertEqual(self.cube.add_aux_factory.call_count, 1)
+        args, _ = self.cube.add_aux_factory.call_args
+        self.assertEqual(len(args), 1)
+        factory = args[0]
+        # Check that the factory has no delta term
+        self.assertEqual(factory.delta, None)
+        self.assertEqual(factory.sigma, mock.sentinel.b)
+        self.assertEqual(factory.surface_air_pressure, mock.sentinel.ps)
+
     def test_formula_terms_ap_missing_coords(self):
-        coordinates = [(mock.sentinel.b, 'b'), (mock.sentinel.ps, 'ps')]
-        self.provides = dict(coordinates=coordinates)
```

Comment:
These lines weren't actually doing anything. Assigning to `self.provides` after `setUp` has run has no effect.